sway/sway-input.5.scd

```diff
@@ -105,10 +105,11 @@ The following commands may only be used in the configuration file.
 *input* <identifier> repeat\_rate <characters per second>
 	Sets the frequency of key repeats once the repeat\_delay has passed.
 
-*input* <identifier> scroll\_button <button\_identifier>
-	Sets button used for scroll\_method on\_button\_down. The button identifier
-	can be obtained from `libinput debug-events`.
-	If set to 0, it disables the scroll\_button on\_button\_down.
+*input* <identifier> scroll\_button 0|button[1-3,8,9]|<event-code-or-name>
```

Comment:
I just went off the `0` value from the man page. Although it looks like the code never actually allowed for `0`.

`disable` would be fine since all valid button events start with `BTN_` and the only other syms we support are the ones starting with `button` for x11 button mappings.

Reply:
Changed it to `disable`